components/script/dom/shadowroot.rs

```diff
@@ -209,7 +224,9 @@ impl ShadowRootMethods for ShadowRoot {
 
     /// https://dom.spec.whatwg.org/#dom-shadowroot-host
     fn Host(&self) -> DomRoot<Element> {
-        DomRoot::from_ref(&self.host)
+        let host = self.host.get();
+        debug_assert!(host.is_some());
```

Comment:
This is pointless, since we unwrap below, just `self.host.get().unwrap()`, or maybe `expect()` if you want to give a nicer message?